### PR TITLE
Cypress/E2E: Fix team and channel assign roles spec

### PR DESCRIPTION
**Pull request by josephbaylon**
#### Summary
- Simplified and refactored change role function
- Fixed test id used

#### Ticket Link
NA

Screen Shot 2020-10-22 at 4 46 49 PM

### CODE REVIEW

File: e2e/cypress/integration/enterprise/ldap_group/team_and_channel_assign_roles_spec.js [hunk 1]
```diff
@@ -136,7 +135,7 @@ describe('System Console', () => {
         });
 
         // # Save the setting and navigate back to page
-        saveAndNavigateBackTo(teamName, 'teams');
+        saveAndNavigateBackTo(teamName, teamDisplayName, 'teams');
```
Comment (saturninoabril) [inline e2e/cypress/integration/enterprise/ldap_group/team_and_channel_assign_roles_spec.js:138]: Test failed on me here.
Screen Shot 2020-10-23 at 6 06 20 PM

It passed when I removed forced click and not using `.then`, like
```javascript
        // # Remove "board" group
        cy.get('.group-row').eq(0).scrollIntoView().should('be.visible').within(() => {
            cy.get('.group-name').should('have.text', groupDisplayName);
            cy.get('.group-actions > a').should('have.text', 'Remove').click();
        });

        // * Assert that the group was removed successfully
        cy.get('#groups-list--body').should('be.visible').contains('No groups specified yet');

        // # Save the setting and navigate back to page
        saveAndNavigateBackTo(teamName, teamDisplayName, 'teams');

        // * Assert that the group was removed successfully
        cy.get('#groups-list--body').scrollIntoView().should('be.visible').contains('No groups specified yet');
```

**Review by saturninoabril — CHANGES_REQUESTED** [review]
LGTM, except for one failing test.

File: e2e/cypress/integration/enterprise/ldap_group/team_and_channel_assign_roles_spec.js (hunk 1 above)
Comment (josephbaylon) [reply e2e/cypress/integration/enterprise/ldap_group/team_and_channel_assign_roles_spec.js:136]: @saturninoabril updated

**Review by srkgupta — APPROVED** [review]
Tested and passed. Thanks @josephbaylon. Changes LGTM

**Pull request merged**